proxy/delete.go

```diff
@@ -39,6 +40,8 @@ func DeleteFunction(gateway string, functionName string) {
 		fmt.Println("Removing old service.")
 	case 404:
 		fmt.Println("No existing service to remove")
+	case 401:
```

Comment:
Please can you update to http.StatusUnauthorized etc? I know these have the numbers in (that's on me..) but let's fix while we're in there.